src/dmd/dsymbolsem.d

```diff
@@ -1530,7 +1530,7 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
     Lnodecl:
         if (pd.decl)
         {
-            pd.error("pragma is missing closing `;`");
+            pd.error("is missing an ending `;`");
```

Comment:
I can't think of any better way to phrase this error. A word such as terminating might be better, but don't jump to that suggestion.

Reply:
I like "terminating".  I think it's consistent with the nomenclature used in programming (terminating null character) and programming language theory (terminals).